products/jse/features/jse_site_owner.feature

```diff
@@ -3,22 +3,22 @@ Feature: Site Owner
   As a Site Owner
   I want to check for my ability to complete my tasks
 
+  Background:
```

Comment:
This background would require that all scenarios use the @api tag. To avoid hilarity later down the road please remove all @api tags from each scenario and place one @api tag at the very first line of this file.